third_party/terraform/website/docs/r/container_cluster.html.markdown

```diff
@@ -372,6 +375,14 @@ for a list of types.
 
 * `maximum` - (Optional) Maximum amount of the resource in the cluster.
 
+The `auto_provisioning_defaults` block supports:
+
+* `oauth_scopes` - (Optional) Scopes that are used by NAP when creating node pools.
```

Comment:
Can you note that `monitoring.write` will always be enabled regardless of user input, and add an infobox ( with `->`) to say that GKE may enable monitoring or logging.write based on `logging_service` and `monitoring_service` values?